Mathlib/MeasureTheory/Integral/Lebesgue.lean

```diff
@@ -1640,6 +1674,77 @@ theorem _root_.IsFiniteMeasure.lintegral_lt_top_of_bounded_to_ennreal {α : Type
   exact ENNReal.mul_lt_top ENNReal.coe_lt_top.ne μ_fin.measure_univ_lt_top.ne
 #align is_finite_measure.lintegral_lt_top_of_bounded_to_ennreal IsFiniteMeasure.lintegral_lt_top_of_bounded_to_ennreal
 
+lemma tendsto_of_lintegral_tendsto_of_monotone {α : Type*} {mα : MeasurableSpace α}
+    {f : ℕ → α → ℝ≥0∞} {F : α → ℝ≥0∞} {μ : Measure α}
+    (hf_meas : ∀ n, AEMeasurable (f n) μ) (hF_meas : AEMeasurable F μ)
```

Comment:
It's not really important, but I expect that this theorem should be true without `hf_meas`.

Reply:
If you don't mind, I'd rather get it in as it is and perhaps come back to improve that later.

Reply:
I have pushed the removal of `hf_meas`.